.claude/skills/akash-bid-matcher/scripts/match_providers.py

```
@@ -0,0 +1,437 @@
+
+
+# ---------- Reporting ----------
+
+def fetch_providers(api_url: str) -> list[dict]:
+    req = urllib.request.Request(
+        api_url,
+        headers={
+            "Accept": "application/json",
+            "User-Agent": "akash-bid-matcher/0.1 (+https://github.com/akash-network/console)",
+        },
+    )
```

Ruff (0.15.11)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)